### ISSUE TITLE
Stopped Site showing up as Healthy

### ISSUE THREAD

**Issue by MattTatoczenko** [ShekharGupta1988/AppServicesSupportCenterPortal#78]
![image](https://cloud.githubusercontent.com/assets/24440502/25451566/379e2c4e-2a88-11e7-83b9-2a844d15c81f.png)

Not sure if this is necessarily something wrong with the detector, but I stopped my site and it's still saying the site is healthy. Browsing to the site gives the 403 stopped page.

![image](https://cloud.githubusercontent.com/assets/24440502/25451604/561fdfaa-2a88-11e7-8fe4-457535957f3c.png)

Should it be calling out that the site is stopped right now and potentially give info based on the type of stop, like Admin Disabled or Over Quota vs just stopped?

**Comment by stevenernst131** [comment]
Yes we can add this. It is healthy in the respect that it is not returning 500 errors
